src/pages/docs/storefront-development/index.tsx

```diff
@@ -1,115 +1,258 @@
-    title: 'Cross-border stores',
-    description:
-      'Go global with multi-currency, multi-language tools and reusable components.',
-    linkTitle: 'See more',
-    linkTo: '/docs/guides/vtex-io-cross-border-stores',
-  },
-  {
-    title: 'B2B stores',
-    description: 'Learn how to implement a B2B store with the Store Framework.',
-    linkTitle: 'See more',
-    linkTo: '/docs/guides/b2b-setup',
-  },
-]
+const FastStoreCMSIcon = (props: IconProps) => {
+  return (
+    <Icon {...props} viewBox="0 0 32 32">
+      <g transform="translate(0 0)">
+        <g
+          strokeLinecap="square"
+          strokeLinejoin="miter"
+          strokeWidth="2"
+          fill="#a1a8b3"
+          stroke="#a1a8b3"
+          strokeMiterlimit="10"
+        >
+          <line
+            data-color="color-2"
+            fill="none"
+            x1="5"
+            y1="9"
+            x2="17"
+            y2="9"
+          ></line>{' '}
+          <polyline
+            data-color="color-2"
+            fill="none"
+            points=" 5,1 5,25 17,25 "
+          ></polyline>{' '}
+          <polygon
+            fill="none"
+            stroke="#4A596B"
+            points="27,15 17,15 17,3 23,3 27,7 "
+          ></polygon>{' '}
+          <polygon
+            fill="none"
+            stroke="#4A596B"
+            points="27,31 17,31 17,19 23,19 27,23 "
+          ></polygon>
+        </g>
+      </g>
+    </Icon>
+  )
+}
 const StorefrontDevelopmentPage: NextPage<Props> = ({ branch }) => {
   const { setBranchPreview } = useContext(PreviewContext)
   setBranchPreview(branch)
   const messages = getMessages()
+  const cardsContent = [
+    {
+      title: 'FastStore',
+      description:
+        'Build high-performance stores based on the Jamstack architecture.',
+      getStarted:
+        'https://developers.vtex.com/docs/guides/faststore/docs-what-is-faststore',
+      links: [
+        {
+          href: 'https://developers.vtex.com/docs/guides/faststore/components-index',
+          title: 'Check the FastStore UI components',
+          icon: <FastStoreComponentsIcon />,
+        },
+        {
+          href: 'https://developers.vtex.com/docs/guides/faststore/using-themes-overview',
+          title: 'Customize your storefront theme',
+          icon: <FastStoreStylesIcon />,
+        },
+        {
+          href: 'https://developers.vtex.com/docs/guides/faststore/headless-cms-overview',
+          title: 'Manage content with the Headless CMS',
+          icon: <FastStoreCMSIcon />,
+        },
+      ],
+    },
+    {
+      title: 'Store Framework',
+      description:
+        'Leverage the VTEX IO infrastructure to build store themes using VTEX IO apps.',
+      getStarted: 'https://developers.vtex.com/docs/guides/store-framework',
+      links: [
+        {
+          href: 'https://developers.vtex.com/docs/guides/vtex-io-documentation-building-a-product-details-page',
+          title: 'Get started with Store Framework blocks',
```

Comment:
Como já temos o botão `Get started` no card, será que pode confundir a pessoa de onde iniciar? Talvez nessa frase deixar algo como: `Explore the Store Framework blocks` ou `Implement Store Framework blocks`...
<img width="398" alt="Screenshot_528" src="https://github.com/vtexdocs/devportal/assets/67270558/94120a19-0cf2-4844-b581-ad29e4810b62">